desktop-widgets/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ set(SUBSURFACE_INTERFACE
 	kmessagewidget.cpp
 	mainwindow.cpp
 	mapwidget.cpp
+	../mobile-widgets/qmlmapwidgethelper.cpp
```

Comment:
yeah, that's REALLY ugly - as is the overflowing commit message.
not enough to ask you to redo this massive pull request, but just saying...